spec/components/candidate_interface/provider_contact_information_component_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::ProviderContactInformationComponent do
+  let(:component) { described_class.new(provider:) }
+  let(:provider) { build(:provider) }
+  let(:result) { render_inline(component) }
+
+  context 'when provider has phone and email' do
+    it 'show provider contact information' do
+      expect(result.text).to include('Contact training provider',
+                                     "Call on #{provider.phone_number}",
+                                     "email at #{provider.email_address}")
+    end
+  end
+
+  context 'when provider has only phone' do
+    let(:provider) do
+      create(:provider, email_address: nil)
+    end
+
+    it 'show provider phone number' do
+      expect(result.text).to include('Contact training provider',
+                                     "Call on #{provider.phone_number}")
```

Comment:
Did you consider adding a not to include "Email at" here? and not to include "Contact" below? Do you think it is worth?

Reply:
I don't think it's worth it but I'm open to being pushed

Reply:
Not a big issue, but I agree with @tomas-stefano. If you don't want to add not include "Email at" and "Call on" I'd remove these two specs as they are already being covered by the first spec. They are only testing if you are not somehow hiding the phone if the email is `nil`, which is an unlikely scenario.

Reply:
I've added it